folly/io/async/test/RequestContextTest.cpp

```diff
@@ -21,6 +21,7 @@
 
 #include <folly/io/async/EventBase.h>
 #include <folly/io/async/Request.h>
+#include <folly/portability/GFlags.h>
```

Comment:
Ideally we should have an alias for this in the GFlags portability header so we continue to work with older versions of GFlags.